domain-ee/ee-ep-merge-app/src/python_src/schema/merge_job.py

```diff
@@ -38,16 +37,14 @@ def __str__(self):
 
 
 class MergeJob(BaseModel):
-    _init_time: ClassVar[datetime] = datetime.now()
-
     job_id: UUID
     pending_claim_id: conint(strict=True)
     ep400_claim_id: conint(strict=True)
     state: JobState = JobState.PENDING
     error_state: JobState | None = None
     messages: list[Any] | None = None
-    created_at: datetime = _init_time
-    updated_at: datetime = _init_time
+    created_at: datetime
+    updated_at: datetime
```

Comment:
I don't know Pydantic super well, but I think we can salvage this rather than go through all that work with every instantiation. What if you import Field from pydantic and do something like

```python
    created_at: datetime = Field(default_factory=datetime.now)
    updated_at: datetime = Field(default_factory=datetime.now)
```

Reply:
Oh nice. I'll play with it and see!

Reply:
Turns out doing it that way makes those two time stamps different. So on job submission it will return different values for those implying that the job has been updated but it hasn't at that point.

Reply:
Oh, yeah I should've thought of that.

Is it still possible to hand-bake it into the model, perhaps by implementing an `__init__` that cleverly provides values for the two fields if they're None (and then calling the super-init)?

Reply:
Figured out a way to set the updated_at to the created_at time on init.